ui/guest/src/react/GuestGlobalStyles.tsx

```diff
@@ -62,184 +63,187 @@ export function GuestGlobalStyles(props: GuestGlobalStylesProps) {
-              }
-            },
-            // endregion
-            '.mce-content-body': {
-              outlineOffset: 5,
-              outline: `2px solid ${palette.blue.tint}`,
-              '&:focus': {
-                outline: `5px solid ${palette.blue.main}`
-              }
-            },
-            // Keyframes
-            '@keyframes craftercms-uploader-mask-animation': {
-              '0%': {
-                opacity: 0.4
               },
-              '50%': {
-                opacity: 0.5
+              // endregion
+              '.mce-content-body': {
+                outlineOffset: 5,
+                outline: `2px solid ${palette.blue.tint}`,
+                '&:focus': {
+                  outline: `5px solid ${palette.blue.main}`
+                }
               },
-              '100%': {
-                opacity: 0.4
-              }
-            },
-            '@keyframes craftercms-placeholder-animation': {
-              '100%': { transform: 'rotate(360deg)' }
-            },
-            '@keyframes craftercms-placeholder-inner-animation': {
-              '0%': { strokeDasharray: '1, 150', strokeDashoffset: '0' },
-              '50%': { strokeDasharray: '90, 150', strokeDashoffset: '-35' },
-              '100%': { strokeDasharray: '90, 150', strokeDashoffset: '-124' }
-            },
-            '@keyframes craftercms-content-tree-locate-animation': {
-              '0%': {
-                transform: 'scaleX(1)'
+              // Keyframes
+              '@keyframes craftercms-uploader-mask-animation': {
+                '0%': {
+                  opacity: 0.4
+                },
+                '50%': {
+                  opacity: 0.5
+                },
+                '100%': {
+                  opacity: 0.4
+                }
               },
-              '50%': {
-                transform: 'scale3d(1.05,1.05,1.05)'
+              '@keyframes craftercms-placeholder-animation': {
+                '100%': { transform: 'rotate(360deg)' }
               },
-              to: {
-                transform: 'scaleX(1)'
+              '@keyframes craftercms-placeholder-inner-animation': {
+                '0%': { strokeDasharray: '1, 150', strokeDashoffset: '0' },
+                '50%': { strokeDasharray: '90, 150', strokeDashoffset: '-35' },
+                '100%': { strokeDasharray: '90, 150', strokeDashoffset: '-124' }
+              },
+              '@keyframes craftercms-content-tree-locate-animation': {
+                '0%': {
+                  transform: 'scaleX(1)'
+                },
+                '50%': {
+                  transform: 'scale3d(1.05,1.05,1.05)'
+                },
+                to: {
+                  transform: 'scaleX(1)'
+                }
               }
-            }
-          },
-          styles ?? {}
-        )}
-      />
+            },
+            styles ?? {}
+          )}
+        />
+        <Global styles={staticGlobalStyles} />
```

Comment:
These styles have more than just the typography. Let's split typography vs other things on the studio Side, so you can import `typograhyStyles` by itself. Also, on the Studio side, where ever those things are use, you'd now need to create 2 `<Global />` blocks to add both.